.azure-pipelines/Windows-CI.yml

```diff
@@ -67,5 +67,5 @@ jobs:
       pip install --quiet -e .[mypy]
       python setup.py typecheck
 
-      git diff --exit-code
+      git diff --exit-code  -- . ':(exclude)onnx/onnx-data.proto' ':(exclude)onnx/onnx-data.proto3'
```

Comment:
on windows platform you dont nee quotation marks
git diff --exit-code  -- . :(exclude)onnx/onnx-data.proto :(exclude)onnx/onnx-data.proto3
should work